common/proto/common.proto

```diff
@@ -7,14 +7,14 @@ message ERC20Tokens {
 }
 
 message ERC20Token {
-  bytes address = 1; // bytes: Address
+  string address = 1; // bytes: Address
```

Comment:
update comment

Reply:
I actually left them in on purpose so we can easily change back after SF adds better support. Even a comment like `bytes: Address` makes it more clear that the string is going to be addresses formatted in hex.